build.cake

```diff
@@ -84,7 +84,8 @@ Task("PushPackage")
             DotNetCoreNuGetPush(package, new DotNetCoreNuGetPushSettings 
             {
                 Source = "https://www.nuget.org/api/v2/package",
-                ApiKey = apiKey
+                ApiKey = apiKey,
+                SymbolApiKey = apiKey
```

Comment:
I think you need to specify -SymbolSource as well : https://docs.microsoft.com/fr-fr/nuget/tools/cli-ref-push

Reply:
SymbolSource="https://www.nuget.org/api/v2/symbolpackage"

Reply:
https://docs.microsoft.com/en-us/nuget/api/symbol-package-publish-resource

Reply:
thanks i forgot this i do it